OracleAccessManagement/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,447 @@
+Building an Oracle Access Management Image using Dockerfile Samples
+==========================================================================
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This Image includes binaries for Oracle Access Management (OAM) Release 12.2.1.4.0 and it has capability to create FMW Infrastructure domain and OAM specific Managed Servers.
+
+***Image***: oracle/oam:<version; example:12.2.1.4.0>
+
+## Prerequisites
+The following prerequisites are necessary before building OAM Docker images:
+
+* A working installation of Docker 18.03 or later
+
+## Prerequisites for OAM on Kubernetes
+
+Refer to the [Prerequisites](https://oracle.github.io/fmw-kubernetes/oam/prerequisites) in the Oracle Access Management Kubernetes documentation.
+
+
+## How to build
+This project offers a sample Dockerfile and scripts to build an Oracle Access Management 12cPS4 (12.2.1.4) image. 
+
+Building your own OAM image involves the following steps:
+
+* Pulling the Oracle JDK Base Image
+* Pulling the Oracle FMW Infrastructure 12.2.1.4 image
+* Downloading the OAM Docker files
+* Downloading the 12.2.1.4.0 Identity Management shiphome and Patches
+* Building the OAM image
+
+
+### Pulling the Oracle JDK (Server JRE) base image
```

Comment:
Replace this with a link to the documentation on how to build the Server JRE image.